tests/network/vmi_multus.go

```diff
@@ -566,6 +568,75 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 				Expect(testErr.ErrStatus.Reason).To(BeEquivalentTo("Invalid"))
 			})
 		})
+
+		Context("MAC Spoof check with Linux bridge CNI plugin interface and custom MAC address.", func() {
+			It("Should allow outbound communication from VM under test - only if original MAC address is unchanged", func() {
+				const (
+					bridgeInterfaceName = "linux-bridge"
+					initialMacAddress   = "50:00:00:00:90:0e"
+					spoofedMacAddress   = "50:00:00:00:80:0a"
+				)
+
+				const (
+					vmUnderTestIPAddress = "10.2.1.1"
+					dummyVMIPAddress     = "10.2.1.2"
+					bridgeSubnetMask     = "/24"
+				)
+
+				linuxBridgeInterfaceWithMACSpoofCheck := v1.Interface{
+					Name: bridgeInterfaceName,
+					InterfaceBindingMethod: v1.InterfaceBindingMethod{
+						Bridge: &v1.InterfaceBridge{},
+					},
+				}
```

Comment:
I think you should refactor `libvmi.InterfaceDeviceWithBridgeBinding()` to support passing in a name as an argument (on a different PR).

Reply:
Addressed on PR #6797